libs/game/game.ts

```diff
@@ -17,6 +17,29 @@ namespace game {
         None
     }
 
+    /**
+     * The available colors for Arcade.
+     * NOTE: If the color pallette is changed, these values will change along with it.
```

Comment:
maybe a pointer here as this is the other place I listed these out https://github.com/microsoft/pxt/blob/stable8.5/webapp/src/components/ImageEditor/sprite/Palette.tsx#L98